go/vt/worker/split_clone.go

```diff
@@ -965,3 +1003,67 @@ func (scw *SplitCloneWorker) getSourceSchema(ctx context.Context, tablet *topoda
 	}
 	return sourceSchemaDefinition, nil
 }
+
+// StatsUpdate receives replication lag updates for each destination master
+// and forwards them to the respective throttler instance.
+// It is part of the discovery.HealthCheckStatsListener interface.
+func (scw *SplitCloneWorker) StatsUpdate(ts *discovery.TabletStats) {
```

Comment:
If ts.Up is false, the tablet is gone from serving. You should not consider it any more. Overall, you need to keep a map from ts.Key to *discovery.TabletStats, and have a function that returns the replication lag you want from that map (second-to-smallest maybe?).